### PR TITLE
add secondsToWaitBeforeKill to user manual

### PR DESCRIPTION
**Pull request by mattnworb**

### CODE REVIEW

**Comment by mattnworb** [comment]
@negz @spotify/helios-team

**Comment by negz** [comment]
LGTM

File: docs/user_manual.md [hunk 1]
```diff
@@ -318,6 +319,18 @@ containers, e.g. `docker run --security-opt <value...>`.
 For more details, see the [Docker
 docs](https://docs.docker.com/reference/run/#security-configuration).
 
+#### secondsToWaitBeforeKill
```
Comment (davidxia) [inline docs/user_manual.md]: I know the other options aren't in order, but can we put this above securityOpt?

Comment (mattnworb) [inline docs/user_manual.md]: sadly I attempted to put this in alphabetical order

Comment (mattnworb) [inline docs/user_manual.md]: fixed

**Pull request merged**